### PR TITLE
Add delete filter funcionallity from Modules Events

### PR DESCRIPTION
**Pull request by Mayons**
This PR includes a refactor on the mapper file of basic modules.
And the remove filter functionallity frorm Module/events

### CODE REVIEW

File: tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature [hunk 1]
```diff
@@ -23,4 +23,28 @@ Feature: Validate that the added filter label is remove after click remove filte
       | NIST                  |
       | TSC                   |
       | Policy Monitoring     |
-      | PCIDSS                |
\ No newline at end of file
+      | PCIDSS                |
+
```
Comment (MauGaP) [inline tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature:27]: delete empty line

File: tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature [hunk 2]
```diff
@@ -1,9 +1,9 @@
-Feature: Validate that the added filter label is remove after click remove filter option
+Feature: Validate that the added filter label is remove after click remove filter option modules Dashboard/Events
    
     As Wazuh user 
     I want to set a new filter 
     in order to manage them
-  
+
    Scenario Outline: The user remove an added filter - Module - Dashboard
```
Comment (MauGaP) [inline tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature:7]: remember the scenario should be the most descriptive of the test we are running

File: tests/frontend/test_kibana_app/cypress/integration/utils/driver.js [hunk 3]
```diff
@@ -138,16 +144,19 @@ export const updateExpiryValueCookies =  () => {
 }
 
 // Function that's return the selector by xpath
```
Comment (MauGaP) [inline tests/frontend/test_kibana_app/cypress/integration/utils/driver.js:146]: I think we don't need this comment

**Review by MauGaP — APPROVED** [review]
just minor changes, great work!

**Review by mauceballos — APPROVED** [review]

**Pull request merged**